Tests/SwiftPrettyPrintTests/Public/CombineExtensionTests.swift

```diff
@@ -142,23 +191,23 @@ final class CombineExtensionTests: XCTestCase {
             receive value: [3, 4]
             receive finished
             """)
+        
+        let recorder2 = StringRecorder()
+        subscribeAndWait(
+            Fail<[Int], TestError>(error: TestError())
+                .prettyPrint(format: .singleline, to: recorder2)
+                .eraseToAnyPublisher()
+        )
+
+        XCTAssert(recorder2.contents.contains("receive failure: TestError(code: 1, message: \"This is the error\")"))
     }
 
     func testMultiline() throws {
         let recorder = StringRecorder()
-        let exp = expectation(description: "")
-        
-        array
-            .publisher
-            .prettyPrint(to: recorder)
-            .handleEvents(receiveCompletion: { _ in
-                exp.fulfill()
-            })
-            .sink { value in }
-            .store(in: &cancellables)
-        
-        wait(for: [exp], timeout: 3)
         
+        subscribeAndWait(
+            array.publisher.setFailureType(to: TestError.self).prettyPrint(to: recorder).eraseToAnyPublisher()
+        )
```

Comment:
I thought the format was inconsistent.

How about a style that breaks the line just before the `.` for consistent of format?

```suggestion
        subscribeAndWait(
            array
                .publisher
                .setFailureType(to: TestError.self)
                .prettyPrint(to: recorder)
                .eraseToAnyPublisher()
        )
```